### ISSUE TITLE
[Feature] Add flows dynamically

### ISSUE THREAD

**Issue by dan2li** [s8sg/goflow#75]
[Feature] Support dynamically appending flows after the flow service started.
Currently, it is only possible to register a workflow before starting the flow service. If you need to register a new workflow after starting the flow service, a restart is required. We  need to provide a more flexible solution in the future.

**Comment by dan2li** [comment]
I'm not quite sure, but it seems like the **task queue** and **consumer** for each flow are initialized in the `StartQueueWorker` function, which is only called once when the flow service is started.

**Comment by dan2li** [comment]
> The idea is to define the workflow in each requests. Even for partial request

Are you suggesting that **after separating flow and workload**, we can attach the complete flow information to each request? That's ok if there is also a method to dynamically register workloads.

**Comment by s8sg** [comment]
I’ll check this over the weekend

**Comment by dan2li** [comment]
Hi @s8sg ，how are things going? :)

**Comment by Lucifer8729** [comment]
hey @s8sg , this is very crucial for a workflow engine

**Comment by s8sg** [comment]
You are welcome to contribute. Also would love to know why it’s crucial for a workflow orchestration. 

I’m not sure if I’ll say goflow is a workflow engine

**Comment by s8sg** [comment]
Duplicate of #66

**Comment by dan2li** [comment]
@s8sg  To clarify, the goal is not to dynamically change the orchestration during runtime, but to register a new flow, which is a different scenario from #66 . When I use goflow to create a workflow platform similar to **Airflow**, which allows users to dynamically define and submit new workflows, this is a necessary function.

**Comment by s8sg** [comment]
@dan2li do you want to register the flow with a new name ?

**Comment by s8sg** [comment]
I might have misunderstood. I think as long as we are not updating the existing workflow I’m fine to dynamically register a new workflow on the same flow service. Reopening the same

**Comment by dan2li** [comment]
Awesome! @s8sg  could you help to review #74 ? :)

**Comment by wangbaobaoget** [comment]
The usage scenario for dynamically creating, stopping, starting, and adding processes is the data center system. When the system accesses data, it needs to dynamically manage all processes, including adding, editing, deleting, starting, and stopping.
